src/lambda/deleteUser.js

```diff
@@ -0,0 +1,34 @@
+import connectToDatabase from './database/dbConnection';
+import { deleteUser } from './database/queries/user';
+import { deleteUserRecords } from './database/queries/rentalRecord';
+
+async function handler(event, context) {
+  // eslint-disable-next-line no-param-reassign
+  context.callbackWaitsForEmptyEventLoop = false;
+  if (event.httpMethod !== 'DELETE') {
+    return {
+      statusCode: 405,
+    };
+  }
+
+  try {
+    const user = JSON.parse(event.body);
```

Comment:
Validate the body and check if valid mongoID

Reply:
I made yup schema for this which should be applied after Simon PR is merged. can be done in a refactor pr